fe/fe-core/src/main/java/com/starrocks/transaction/TransactionState.java

```diff
@@ -469,6 +469,12 @@ public void updatePublishTaskFinishTime() {
         this.publishVersionFinishTime = System.currentTimeMillis();
     }
 
+    // for test
+    public long getPublishTaskFinishTime() {
```

Comment:
```suggestion
   long getPublishTaskFinishTime() {
```

remove the access declaration, so the interface can be accessible from within the package and can't be accessible outside the package. just good enough for unit test